src/SmartComponents/Recs/List.js

```diff
@@ -45,20 +50,22 @@ const List = () => {
   }, [chrome, intl]);
 
   const [activeTab, setActiveTab] = useState(
-    pathname === '/recommendations/pathways' ? 1 : 0
+    pathname === '/recommendations/pathways'
+      ? PATHWAYS_TAB
+      : RECOMMENDATIONS_TAB
   );
   const changeTab = (tab) => {
     setActiveTab(tab);
-    navigate(tab === 1 ? '/recommendations/pathways' : '/recommendations');
+    navigate(
```

Comment:
This causes a `navigate` function is undefined error. The problem is likely, because the `const { navigate } = useInsightsNavigate();` on line 40 above, is wrong. It should be `const navigate = useInsightsNavigate();` instead.